app/views/works/_edit_javascript.html.erb

```diff
@@ -354,6 +385,21 @@
       }
     }
 
+    // Load any existing related objects into the edit form. 
+    // If there are any related objects they should appear in hidden <span> tags.
+    if ($(".related-object-data").length ==0) {
+      // Add an empty related object for the user to fill it out
+      var num = incrementCounter("#related_object_count");
+      addRelatedObjectHtml(num, "", "", "");
+    } else {
+      // Add existing related objects for editing
+      var related_objects = $(".related-object-data");
+      for(i = 0; i < related_objects.length; i++) {
+        var related_object = related_objects[i];
+        addRelatedObjectHtml(related_object.dataset.num, related_object.dataset.relatedIdentifier, related_object.dataset.relatedIdentifierType, related_object.dataset.relationType);
```

Comment:
```suggestion
        const {num, relatedIdentifier, relatedIdentifierType, relationType} = related_object.dataset
        addRelatedObjectHtml(num, relatedIdentifier, relatedIdentifierType, relationType);
```

Reply:
Sorry, my js isn't the greatest. I don't understand what is being suggested here.